src/services/print.js

```diff
@@ -360,12 +361,19 @@ ngeo.Print.prototype.encodeVectorLayer_ = function(arr, layer, resolution) {
       if (goog.isNull(geojsonFeature.properties)) {
         geojsonFeature.properties = {};
       }
+      var geometry = feature.getGeometry();
+      var geometryType;
+      if (goog.isDef(geometry) && !goog.isNull(geometry)) {
+        geometryType = geometry.getType();
+      } else {
+        geometryType = '';
+      }
```

Comment:
I think we can just skip the feature if it doesn't have a geometry. And note that you can use `goog.isDefAndNotNull`.